editor/src/core/model/element-template-utils.spec.ts

```diff
@@ -362,3 +364,117 @@ export const TestComponent = (props) => {
     expect(result).toEqual(true)
   })
 })
+
+describe('componentHonoursPropsPosition', () => {
```

Comment:
could you insert a few test cases for false results? just to make sure the function doesn't accidentally regress into "always true"

Reply:
Additional test cases added in 6744435ba.